doc/source/io.rst

```diff
@@ -2186,6 +2237,7 @@ A few notes on the generated table schema:
 
   .. ipython:: python
 
+
```

Comment:
I guess this is giving an error, but that will be fixed in flake8-rst, as the error is wrong. So just one blank line please.